vulnfeeds/cmd/debian/main.go

```diff
@@ -97,9 +97,18 @@ func updateOSVPkgInfos(pkgName string, cveId string, releases map[string]Release
 	if value, ok := osvPkgInfos[cveId]; ok {
 		pkgInfos = value
 	}
-	for releaseName, release := range releases {
+
+	// Sorts releases to ensure pkgInfos remain consistent between runs.
+	var keys []string
+	for name := range releases {
```

Comment:
Can you sort this by the release order? E.g. have debian 10 before debian 11 before debian 12. I think you can just use the `debianReleaseMap` as the key to sort by.